[Activity] Document escrow disk encryption key activities (#50236)
fleetdm/fleet: docs/Contributing/reference/audit-logs.md

```

## enabled_macos_disk_encryption

> > **Deprecated:** Use platform based `edited_disk_encryption_settings` instead.

Reviewer: ```suggestion
> **Deprecated:** Use `edited_disk_encryption_settings` instead.
```

```

## disabled_macos_disk_encryption

> > **Deprecated:** Use platform based `edited_disk_encryption_settings` instead.

Reviewer: ```suggestion
> **Deprecated:** Use `edited_disk_encryption_settings` instead.
```

}
```

## edited_disk_encryption_settings

Generated when a user edits disk encryption settings for hosts on a fleet (or unassigned hosts).

This activity contains the following fields:
- "fleet_id": The ID of the fleet, `null` if it applies to hosts that are not in a fleet ("Unassigned").
- "fleet_name": The name of the fleet, `null` if it applies to devices that are not in a fleet ("Unassigned").

> - "platform": The platform of the fleet. Options are "macos", "windows", or "linux".

Reviewer: ```suggestion
- "platform": The platform that disk encryption settings were updated for. Options are "macos", "windows", or "linux".
```

}
```

> ## edited_disk_encryption_settings_apple

Reviewer: We could also potentially combine these into 1 activity with a `platform` key. @melpike wdyt?

Author: @rachaelshaw, updated! lmk what you think.
